pkg/rulefmt/rulefmt.go

```diff
@@ -176,9 +176,17 @@ func checkOverflow(m map[string]interface{}, ctx string) error {
 
 // Parse parses and validates a set of rules.
 func Parse(content []byte) (*RuleGroups, []error) {
+	return parse(content, "")
+}
+
+func parse(content []byte, filename string) (*RuleGroups, []error) {
 	var groups RuleGroups
 	if err := yaml.Unmarshal(content, &groups); err != nil {
-		return nil, []error{err}
+		if filename == "" {
+			return nil, []error{err}
+		} else {
+			return nil, []error{fmt.Errorf("filename=%s, err=%v", filename, err)}
```

Comment:
Will this make an error like: `err="filename="foo" err="yaml: line 17: found unexpected end of stream""`?